modules/lwjgl/core/src/main/java/org/lwjgl/system/SharedLibraryLoader.java

```diff
@@ -36,9 +35,7 @@ final class SharedLibraryLoader {
 
     private static final Lock EXTRACT_PATH_LOCK = new ReentrantLock();
 
-    @GuardedBy("EXTRACT_PATH_LOCK")
```

Comment:
This is the only use of JSR305 annotations that can't be replaced with JSpecify. We _could_ keep JSR305 just for that but I figured it's not worth it.